components/meetstream_ai/actions/create-bot/create-bot.mjs

```diff
@@ -0,0 +1,74 @@
+import meetstreamAi from "../../meetstream_ai.app.mjs";
+
+export default {
+  key: "meetstream_ai-create-bot",
+  name: "Create Bot",
+  description: "Creates a new bot instance to join a meeting. [See the documentation](https://vento.so/view/35d0142d-f91f-47f6-8175-d42e1953d6f1?utm_medium=share)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    meetstreamAi,
+    meetingLink: {
+      type: "string",
+      label: "Meeting Link",
+      description: "The link to the meeting where the bot should join",
+    },
+    botName: {
+      type: "string",
+      label: "Bot Name",
+      description: "The name of the bot",
+      optional: true,
+    },
+    audioRequired: {
+      type: "boolean",
+      label: "Audio Required",
+      description: "Whether audio is required",
+      optional: true,
+    },
+    videoRequired: {
+      type: "boolean",
+      label: "Video Required",
+      description: "Whether video is required",
+      optional: true,
+    },
+    liveAudioRequired: {
+      type: "object",
+      label: "Live Audio Websocket URL",
+      description: "Specify websocket_url for live audio streaming",
+      optional: true,
+    },
+    liveTranscriptionRequired: {
+      type: "object",
+      label: "Live Transcription Webhook URL",
+      description: "Specify webhook_url for live transcription",
+      optional: true,
+    },
```

Comment:
Should these be of type `boolean`? Or are they meant to be a `string` containing the webhook_url? If they're meant to be `objects`, can we put an example in the descriptions, and then parse them in the request?